client/scraper/metadata.py

```diff
@@ -73,6 +75,8 @@ def parse_content(self, response):
 
     def to_json(self):
         json_data = json.dumps(self.prop_map)
+        if config.CACHE_DATA:
```

Comment:
I don't think cache should be in `to_json`. We should instead put it in a new function like `cache_json_data` and call it in `api_handler`